packages/ui/src/components/msg/governance/deposit_proposal/index.tsx

```diff
@@ -5,8 +5,12 @@ import { formatNumber, formatToken } from '@/utils/format_token';
 import { PROPOSAL_DETAILS } from '@/utils/go_to_page';
 import Typography from '@mui/material/Typography';
 import { Trans, useTranslation } from 'next-i18next';
-import Link from 'next/link';
-import { FC, useMemo } from 'react';
+import Link, { LinkProps } from 'next/link';
+import { FC } from 'react';
+
+const LinkText = ({ href, children }: React.PropsWithChildren<LinkProps>) => (
```

Comment:
Which style do you prefer: the current one or the following?
```
import { FC, PropsWithChildren } from 'react';
... : PropsWithChildren<LinkProps>
```
I like the shorter code, but I know most people use the `React.` prefix

Reply:
ok modified the style [e7defdb](https://github.com/forbole/big-dipper-2.0-cosmos/pull/1175/commits/e7defdbc2e1479ec48261f78ba3f61c7952e5a1f)